package.json

```diff
@@ -45,24 +45,23 @@
     "@nguniversal/express-engine": "^6.0.0",
     "@nguniversal/module-map-ngfactory-loader": "^6.0.0",
     "core-js": "^2.4.1",
-    "rxjs": "6.0.0",
+    "rxjs": "^6.2.2",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.6.0",
     "@angular/cli": "6.0.0",
-    "@angular/compiler-cli": "6.0.0",
-    "@angular/language-service": "6.0.0",
+    "@angular/compiler-cli": "^6.0.0",
```

Comment:
these should always match the `@angular/package` versions and not the cli version